cmd/argocd/commands/app.go

```diff
@@ -551,6 +563,31 @@ func NewApplicationListCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 	return command
 }
 
+func formatConditionsSummary(app argoappv1.Application) string {
+	typeToCnt := make(map[string]int)
+	for i := range app.Status.Conditions {
+		condition := app.Status.Conditions[i]
+		if cnt, ok := typeToCnt[condition.Type]; ok {
+			typeToCnt[condition.Type] = cnt + 1
+		} else {
+			typeToCnt[condition.Type] = 1
+		}
+	}
+	items := make([]string, 0)
+	for cndType, cnt := range typeToCnt {
+		if cnt > 1 {
+			items = append(items, fmt.Sprintf("%s(%d)", cndType, cnt))
+		} else {
+			items = append(items, cndType)
+		}
+	}
+	summary := "No conditions"
+	if len(items) > 0 {
+		summary = strings.Join(items, ";")
```

Comment:
Nit: can we use comma instead of semicolon? kubectl does this when there are more values (e.g. with labels)